backend/src/routes/user.js

```diff
@@ -1,85 +1,18 @@
 const express = require("express");
-var multer = require("multer");
-var fs = require("fs");
-var path = require("path");
-
-var { authService } = require("../services");
-var { isAuth, attachCurrentUser } = require("../middleware");
+const multer = require("multer");
 
 const router = express.Router();
-const upload = multer();
-
-router.get("/:userId", isAuth, attachCurrentUser, async (req, res) => {
-  if (req.user.id !== req.params.userId) return res.sendStatus(401);
-  const user = await req.context.models.User.findById(req.params.userId);
-  const userData = user.dataValues;
 
-  // Delete extraneous data
-  delete userData.createdAt;
-  delete userData.updatedAt;
-
-  res.send(userData);
-});
-
-router.get("/avatar/:userId", isAuth, attachCurrentUser, (req, res) => {
-  let filePath = path.join(__dirname, `../../data/${req.params.userId}.jpg`);
-  if (fs.existsSync(filePath)) res.sendFile(filePath);
-  else res.sendStatus(404);
-});
+const userController = require('../controllers/user');
+const { isAuth, attachCurrentUser } = require("../middleware");
```

Comment:
Would it be possible to move these below the library imports? It's a little weird to have local imports follow actual logic.

The way I was trying to organize it (loosely) was:

- library imports (express, etc.)
- local imports (our routes, models, middleware, etc.)
- Then the code and logic begins

Finishing your imports, then starting imports again is a little confusing to read